functions.py

```diff
@@ -2,39 +2,22 @@
 from PIL import Image
 import os, sys
 
+class ImageResizer:
 
-def cut_image():
-    print '''
-    =====================
-    CUT IMAGE
-    =====================
-    1. File
-    2. Folder
-    3. Close
-    =====================
-    '''
-    choice = raw_input(">> ")
-    if choice == "1":
-        directory = raw_input('Directory to file: ')
-        width = input('Width: ')
-        height = input('Height: ')
-        name = raw_input('Save as: ')
+    def cut_image(self, directory, width, height, name):
         im = Image.open(directory)
         im = im.resize((width, height))
         if not os.path.exists('resized_images'):
             os.mkdir('resized_images')
         try:
-            print "Size: " + str(width) + " x " + str(height)
             im.save('resized_images/'+name)
             print "- " + name + " : " + "DONE"
             raw_input("Saved in resized_images.")
+            print "Size: " + str(width) + " x " + str(height)
         except:
             raw_input('Error..')
-
-    elif choice == "2":
-        f_directory = raw_input('Directory to folder: ')
-        width = input('Width: ')
-        height = input('Height: ')
+            
+    def cut_image_folder(self, f_directory, width, height):
```

Comment:
`f_directory` name is not expressive. We dont use one letter name shortcuts. Maybe u should use some name like `images_folder_path` - cause its a path (string) to folder with images.
And similar in function above -> `def cut_image(self, directory, width, height, name):` 
Maybe `image_path` instead of  `directory` would be better. 

I understand that this function is to cut a set of images in the folder. If so... name of the function should be sth like `cut_images_in_folder`, not `cut_image_folder` which in direct translation means 'tnij folder obrazu' which not makes sense at all :)